recipes-bsp/firmware-pack/fiptool-native.inc

```diff
@@ -0,0 +1,24 @@
+SECTION = "bootloaders"
+SUMMARY = "Fiptool from Trusted Firmware-A"
+LICENSE = "MIT"
+
+inherit native
+
+DEPENDS = "openssl-native"
+
+S = "${WORKDIR}/git"
+
+do_configure () {
+    sed -i '/^LDLIBS/ s,$, \$\{BUILD_LDFLAGS},' ${S}/tools/fiptool/Makefile
```

Comment:
Here is my explains:
- Sed search for the line started with LDLIBS and add ${BUILD_LDFLAGS} at the end of the line.
- Do not use Make args to replace it because add Make args just apply for Makefile in the ${S} folder.
- In this situation , I want to apply changes in Makefile at path: ${S}/tools/fiptool/Makefile.